boards/arm/mimxrt1170_evk/doc/index.rst

```diff
@@ -121,7 +123,10 @@ The MIMXRT1170 SoC has four pairs of pinmux/gpio controllers.
 +---------------+-----------------+---------------------------+
 | GPIO_AD_25    | LPUART1_RX      | UART Console              |
 +---------------+-----------------+---------------------------+
-
+| GPIO_LPSR_00  | CAN3_TX         | flexcan                   |
++---------------+-----------------+---------------------------+
+| GPIO_LPSR_01  | CAN3_RX         | flexcan                   |
++---------------+-----------------+---------------------------+
 
```

Comment:
Why are there no changes to pinmux.c file?

Reply:
hi,
    Because there is already a pinmux configuration of these pins inside the baseline project:
![image](https://user-images.githubusercontent.com/49855018/124847139-d7b1ac80-dfcc-11eb-8e3e-236005148f15.png)

Regards,
    Crist